tests/apps/sequencing_metrics_parser/test_sequencing_metrics_calculator.py

```diff
@@ -0,0 +1,43 @@
+from cg.apps.sequencing_metrics_parser.sequencing_metrics_calculator import (
+    q30_ratio,
+    average_quality_score,
+    yield_in_megabases,
+    pass_filter_ratio,
+    perfect_reads_ratio,
+    average_clusters_per_lane,
+)
+
+
+def test_calculate_yield_in_megabases():
```

Comment:
Add doc str to tests

Reply:
I do think they are self documenting, but added a doc string anyway to each test